packages/blockchain/src/db/operation.ts

```diff
@@ -13,6 +13,8 @@ import {
 
 import type { CacheMap } from './manager.js'
 
+export type PutOpts = { canonical?: boolean; parentTd?: bigint }
```

Comment:
Can you move this over to `types.ts` and give both option some docs?
(also for (my) understanding)

Reply:
This is still one of the crucial tasks so that we can work on some better understanding and documentation for these options and eventually come to better names.

So, if we have the definitions as comments over the options, this can be reviewed and alternatives/improvements can be suggested.